src/programs/TopicManager.ts

```diff
@@ -35,6 +44,55 @@ export default async function TopicManager(message: Message) {
       message.channel.send(new MessageAttachment(INKTOBER_IMAGE_URL));
       break;
 
+    case "trends":
+      const topicRepo = await TopicManagerRepo();
+      if (!addition) {
+        const currentTrend = await topicRepo
+          .createQueryBuilder("trend")
+          .select()
+          .where("trend.channel = :channel", { channel: "trends" })
+          .orderBy("trend.id", "DESC")
+          .limit(1)
+          .getOne();
+        if (currentTrend) {
+          message.reply(`Current Trend: ${currentTrend.topic}`);
+        } else {
+          message.reply("There are no current trends, create one!");
+        }
+      } else {
+        if (!isAuthorModerator(message)) {
+          message.react("👎");
+          return;
+        }
+        const cleanMessage = message.cleanContent.split(/\s+/);
+        cleanMessage.shift();
+        const joinedMsg = cleanMessage.join(" ");
+        const newTopic = topicRepo.create({
+          topic: joinedMsg,
+          channel: "trends",
+          lastUsed: new Date(),
+        });
+
+        try {
+          await topicRepo
+            .createQueryBuilder()
+            .insert()
+            .into(TopicManagerEntity)
+            .values([newTopic])
+            .execute()
```

Comment:
I *think* this can be shortened to 
```suggestion
            .save(newTopic)
```